app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parsed_command = command.split()
+    path = parsed_command[-1].split("/")
+    directory = ""
+
+    for i in range(len(path) - 1):
```

Comment:
Your code does not handle case when destination path is directory: `mv file.txt dir1/dir2/`